app/helpers/template_paths_helper.rb

```diff
@@ -0,0 +1,14 @@
+module TemplatePathsHelper
+  def template_path_for(type, route = 'path')
```

Comment:
I would be ok we supporting just the path for now, and adding the second method once is really needed (I'm not fond of code for 'just in case' situations). If we needed both path and url, then having two separate methods, that use the same helper for getting the prefix, as more straightforward. Also, what's your plan extending this form plugins, such as remote execution job templates?